http/1.5/modules/ROOT/pages/http-troubleshooting.adoc

```diff
@@ -63,15 +68,15 @@ For example, the following configuration will send a `Host: somehost.com` header
 
 Without the explicit header configuration, an autogenerated `Host: somehost.com:8081` would be sent.
 
-=== HTTP Encoding Support
+== Troubleshoot HTTP Encoding
```

Comment:
```suggestion
== Correct HTTP Encoding
```

Reply:
I suggest similar rewording here, to avoid "there is" (per style ref) and to minimize:

To obtain better performance when working with streamed data, HTTP Connector sends...encoding when the size of the data being sent is unknown. However, some HTTP clients and servers...fail.

Report such failures to the server or client owner to correct. Meanwhile, work around the issue by manually modifying the *Listener* and *Request* operations to output `Content-Length` encoded bodies.